grafana/overlays/nerc-ocp-obs/externalsecrets/oauth-client-secret.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: oauth-client-secret
+  namespace: grafana
+spec:
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: oauth-client-secret
+  data:
+  - secretKey: GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET
+    remoteRef:
+      key: nerc/nerc-ocp-infra/dex/dex-clients
```

Comment:
This looks incorrect (as do all the following entries). You won't be able to access `nerc/nerc-ocp-infra` secrets from the obs cluster, nor will you be able to access dex.

Reply:
Thanks! Then we will to look into that again.
We tried to realize this idea
https://github.com/OCP-on-NERC/docs/tree/main/architecture/observability#architecture-diagram
but maybe had a wrong approach in doing so/making it work.